### PR TITLE
focusin: Added missing signature

### PR DESCRIPTION
**Pull request by AurelioDeRosa**

### CODE REVIEW

File: entries/focusin.xml [hunk 1]
```diff
@@ -19,6 +19,9 @@
       <argument name="eventObject" type="Event" />
     </argument>
   </signature>
+  <signature>
+    <added>1.0</added>
+  </signature>
   <longdesc>
     <p>This method is a shortcut for <code>.on('focusin', handler)</code>.</p>
```
Comment (arthurvr) [inline entries/focusin.xml]: > This method is a shortcut for .on( "focusin", handler ) in the first two variations, and .trigger( "focusin" ) in the third.

Comment (arthurvr) [inline entries/focusin.xml]: Also please fix the style guide violations in `.on('focusin', handler)` :)

**Comment by AurelioDeRosa** [comment]
Should be good now @arthurvr

**Comment by arthurvr** [comment]
lgtm

**Pull request closed without merging**